apps/api/src/modules/job/job.service.spec.ts

```diff
@@ -267,4 +268,37 @@ describe('JobService', () => {
       })
     })
   })
+
+  describe('postJob', () => {
+    const MOCK_TITLE = 'test title'
+    const MOCK_DESCRIPTION = 'test description'
+    const MOCK_STATUS = JobStatus.OPEN
+    const MOCK_ROLE = 'test role'
+    const MOCK_AGE = 20
+    const MOCK_ACTORCOUNT = 1
+    const MOCK_WAGE = 10000
+    const MOCK_DEADLINE = new Date()
+    const MOCK_SHOOTING = mock('shooting').get(3)
+    const MOCK_CASTING_ID = 1
+
+    const MOCK_JOB = {
+      ...mock('job')
+        .omit(['castingId', 'createdAt', 'updatedAt', 'jobId', 'status'])
+        .get(),
+      castingId: MOCK_CASTING_ID,
+      shooting: mock('shooting').get(3),
+    }
+
+    describe('normal behavior', () => {
+      it('should create the job successfully', async () => {
+        jest.spyOn(repository, 'createJob').mockResolvedValue()
+        jest.spyOn(service, 'createJob').mockResolvedValue()
```

Comment:
Why did you mock return service? If you mock return that how can you test whether the return value is correct or not?